src/main/java/seedu/address/logic/commands/CountryFilterCommand.java

```diff
@@ -12,6 +12,12 @@
 public class CountryFilterCommand extends Command {
 
     public static final String COMMAND_WORD = "country filter";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all clients whose country matches "
+            + "the specified country-code and displays them as a list with index numbers.\n"
+            + "Parameters: "
+            + PREFIX_COUNTRY + "COUNTRY_CODE\n"
```

Comment:
Probably can be on the same line for clarity

Reply:
Done